sw/applications/example_asm/add.s

```diff
@@ -0,0 +1,6 @@
+    .section .text
+    .globl add
+
+add:
```

Comment:
can you please call it "add_asm_function" ? easier to find it in the object dump